File: src/main/scala/Coucou.scala
import java.util.ArrayList;
import scala.collection.JavaConversions._

object geeks
{
  def main(args: Array[String])
  {
    val students = new ArrayList[String]("Linda", "Alberto", "Federico", "Luca")

    for (student <- students)
    {
      println("Salut" + student)
    }
  }
}

File: src/main/java/main/Coucou.java
import java.util.ArrayList;
import java.util.List;

public class CreateArrayList
{
    public static void main(String[] args)
    {
        List<String> students = new ArrayList<>("Linda", "Alberto", "Federico", "Luca");
        for (String student : students)
        {
            System.out.println("Salut" + student);
        }
    }
}